op-node/Makefile

```diff
@@ -26,6 +26,8 @@ fuzz:
 	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzL1InfoRoundTrip ./rollup/derive
 	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzL1InfoAgainstContract ./rollup/derive
 	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzUnmarshallLogEvent ./rollup/derive
+	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzParseFrames ./rollup/derive
```

Comment:
Whats the deal with NOTAREALTEST? So that it doesn't run any tests and just fuzzes? Seems like a hack, there is really no way to just run the fuzz?

Reply:
Yep, the `go test` command can only run one fuzz at a time & also tries to run all the tests unless you provide this flag.